src/js/osweb/system/transfer.js

```diff
@@ -66,7 +66,7 @@ export default class Transfer {
 		// Osexp files can be basic text files, or be a zip file.
 		// Check if mimetype of supplied file is known, and load it accordingly.
 		if (this._runner._mimetype.indexOf('text/') != -1) {
-			contents = this._readRemoteOsexpText(url);
+			var contents = this._readRemoteOsexpText(url);

```

Comment:
Using var's in Javascript (ES6) is deprecated and discouraged, because it does strange things with the scope.
Better nowadays is to use `const` if the variable will never be reassigned, or `let` if this does (need to) happen at a later stage.